build.proj

```diff
@@ -131,16 +130,11 @@
     <!-- Build the Microsoft.Azure.Build.Tasks project -->
     <Exec Command="dotnet publish $(RepoTools)BuildPackagesTask/Microsoft.Azure.Build.Tasks/Microsoft.Azure.Build.Tasks.csproj -c $(Configuration)" />
 
-    <!-- Get all of the files changed in the given pull request -->
-    <FilesChangedTask RepositoryOwner="Azure" RepositoryName="azure-powershell" PullRequestNumber="$(PullRequestNumber)"  OutputFile="$(RepoArtifacts)FilesChanged.txt">
```

Comment:
If we remove this task it also means we lose the ability to do a "filtered" build locally - because we now rely on a pipeline step to get changed files. Is that OK? Are there any scenarios we want to do this locally?

Reply:
If the answer is OK, also consider removing the source code of `FilesChangedTask`

Reply:
I would say this task is not needed for local development, because if we have a PR number it means we have a PR opened, it also means there is a PR validation triggered and 'changedFIles.txt' is downloadable. Placing this file under artifact is enough for local usage.